examples/heating_and_cooling/src/run_case.py

```diff
@@ -42,6 +42,13 @@ def __init__(self, *args, **kwargs):
                 self.__indx_max_peak = None
                 self.__day_steps = 5
 
+            # def post(self):     # Make sure the file has access to this method. Probably with a from import at the top.
+            #     super().post()
+            #     self._write_updated_esdl(
+            #         self._ESDLMixin__energy_system_handler.energy_system,
+            #         optimizer_sim=False,
+            #     )
+
```

Comment:
These comment blocks I just kept them there for now while I was developing for easy restoring in case I broke something. I will clean it up and delete them before the final commit for review.

Reply:
I will probably just remove it before the final commit. I just had it there to output optimized esdl's during debugging.